install_config/install/advanced_install.adoc

```diff
@@ -1181,6 +1181,184 @@ xref:../../install_config/persistent_storage/dynamically_provisioning_pvs.adoc#i
+endif::[]
+ifdef::openshift-origin[]
+openshift_service_catalog_image_prefix=openshift/origin-
+openshift_service_catalog_image_version=latest
+endif::[]
+----
+
+When the service catalog is enabled, the web console shows the updated landing
+page but still uses the normal image stream and template behavior. The Ansible
+service broker is also enabled; see
+xref:configuring-ansible-service-broker[Configuring the Ansible Service Broker]
+for more details. To deploy and registry the template service broker (TSB), see
+xref:configuring-template-service-broker[Configuring the Template Service Broker].
+
+[[configuring-ansible-service-broker]]
+=== Configuring the Ansible Service Broker
+
+[NOTE]
+====
+Enabling the Ansible service broker is a Technology Preview feature only.
+ifdef::openshift-enterprise[]
+Technology Preview features are not
+supported with Red Hat production service level agreements (SLAs), might not be
+functionally complete, and Red Hat does not recommend to use them for
+production. These features provide early access to upcoming product features,
+enabling customers to test functionality and provide feedback during the
+development process.
+
+For more information on Red Hat Technology Preview features support scope, see
+https://access.redhat.com/support/offerings/techpreview/.
+endif::[]
+====
+
+If you have xref:enabling-service-catalog[enabled the service catalog], the
+xref:../../architecture/service_catalog/ansible_service_broker.adoc#arch-ansible-service-broker[Ansible service broker] (ASB) is also enabled.
+
+The ASB deploys its own etcd instance separate from the etcd used by the rest of
+the {product-title} cluster. The ASB's etcd instance requires separate storage
+using persistent volumes (PVs) to function. If no PV is available, etcd will
+wait until the PV can be satisfied. The ASB application will enter a `CrashLoop`
+state until its etcd instance is available.
+
+[NOTE]
+====
+The following example shows usage of an NFS host to provide the required PVs,
+but
+xref:../../install_config/persistent_storage/index.adoc#install-config-persistent-storage-index[other persistent storage providers] can be used instead.
+====
+
+Some Ansible playbook bundles (APBs) may also require a PV for their own usage.
+Two APBs are currently provided with {product-title} 3.6: MediaWiki and
+PostgreSQL. Both of these require their own PV to deploy.
+
+To configure the ASB:
+
+. In your inventory file, add `nfs` to the `[OSEv3:children]` section to enable
+the `[nfs]` group:
++
+----
+[OSEv3:children]
+masters
+nodes
+nfs
+----
+
+. Add a `[nfs]` group section and add the host name for the system that will
+be the NFS host:
++
+----
+[nfs]
+master1.example.com
+----
+
+. In addition to the settings from xref:enabling-service-catalog[Enabling the
+Service Catalog], add the following in the `[OSEv3:vars]`
+section:
++
+----
+openshift_hosted_etcd_storage_kind=nfs
+openshift_hosted_etcd_storage_host=<nfs_host> <1>
```

Comment:
After BZ#1467574 is fixed, now installer supports two kinds for nfs export dir, one is existing nfs export dir, another one is creating new nfs server and new nfs export dir on this server.
In this example, once there is nfs group is defined in inventory host file, we should choose new nfs export dir for etcd pv. If user want to use existing nfs export dir, that will make defining nfs group meaningless.